### ISSUE TITLE
No method found matching xxxx

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#357]
I get the following printouts on my terminal when I start Ginga:

```
No method found matching 'gs_swipe'
No method found matching 'ms_draw'
No method found matching 'kp_poly_add'
No method found matching 'kp_edit_del'
No method found matching 'ms_cursor'
No method found matching 'ms_none'
No method found matching 'kp_poly_del'
No method found matching 'ms_wheel'
No method found matching 'gs_tap'
```

It is emitted by `ImageViewBindings.setup_settings_events()` method in `Bindings.py`, in the following logic block:

```python
            # Register for this symbolic event if we have a handler for it
            try:
                cb_method = getattr(self, name)

            except AttributeError:
                self.logger.warning("No method found matching '%s'" % (name))
                cb_method = None
```

And then farther down the codes, I see one of the offending callback name is explicitly commented out:

```python
    ## def ms_draw(self, viewer, event, data_x, data_y):
    ##     return False
```

So, I gather there are no methods defined for the callbacks in the message but they are done on purpose. And there is nothing a user can do about it, so why emit such a warning?

**Comment by ejeschke** [comment]
@pllim, I think at least in some of these cases there is no method in `Bindings` class, but we still want the event to be registered, because some plugins will subscribe for the event later.  I do agree that these messages are annoying and should be eliminated, either by supplying a method that does nothing or by simply eliminating the warning.

I'll take a closer look at it when I get back from vacation in a few days, but if you want to propose something with a PR please feel free.

**Comment by pllim** [comment]
No problem, it's not urgent.

p.s. Does vacation mean you walk 10 feet down to the beautiful Hawaiian beach? :smile:

**Comment by ejeschke** [comment]
> p.s. Does vacation mean you walk 10 feet down to the beautiful Hawaiian beach?
No that's just called "after work"

**Comment by ejeschke** [comment]
@pllim, this should be addressed in #360

**Comment by ejeschke** [comment]
@pllim, is this solved for you by #360?  Can I close it?

**Comment by pllim** [comment]
Yes. Thanks!
